### PR TITLE
Update csi-proxy to use beta version

### PR DESCRIPTION
**Pull request by jingxu97**
This PR update csi-proxy version, remove diskutil.exe and set up log
file

**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by jingxu97** [comment]
cc @pjh @yliaog

**Comment by jingxu97** [comment]
@mtaufen could you pls approve?

File: cluster/gce/config-common.sh [hunk 1]
```diff
@@ -162,4 +162,4 @@ export WINDOWS_INFRA_CONTAINER="gcr.io/gke-release/pause-win:1.2.1"
 # Storage Path for csi-proxy. csi-proxy only needs to be installed for Windows.
 export CSI_PROXY_STORAGE_PATH="https://storage.googleapis.com/gke-release/csi-proxy"
 # Version for csi-proxy
-export CSI_PROXY_VERSION="v0.1.0-gke.1"
\ No newline at end of file
+export CSI_PROXY_VERSION="v0.2.0-gke.0"
```
Comment (mtaufen) [inline cluster/gce/config-common.sh:165]: nit: Add a newline at the end of the file. It's technically a POSIX requirement (and why GitHub puts the (-) symbol on the diff).

Comment (mtaufen) [reply cluster/gce/config-common.sh:165]: approved so you can do this in a follow up if you want

Comment (jingxu97) [reply cluster/gce/config-common.sh:165]: got it. thanks!

**Pull request merged**